### DISCUSSION TITLE
SSL certificate problem

### DISCUSSION THREAD

**Discussion by ocattau** [RobertsLab/resources#1183] [Q&A] [answered]
I am trying to run Blast on Jupyter notebooks, but when I try to open the code I was given here: https://owl.fish.washington.edu/halfshell/genomic-databank/hemat_transcriptome_v1.5.fasta using 

!curl https://owl.fish.washington.edu/halfshell/genomic-databank/hemat_transcriptome_v1.5.fasta I keep getting an SSL certificate problem. This is probably because I haven't figured out how to tell the Jupyter command line how to log into a uw account.

**Comment by kubu4** [comment, marked as answer]
Change your command to one of the following:

`!curl http://owl.fish.washington.edu/halfshell/genomic-databank/hemat_transcriptome_v1.5.fasta`

OR

`!curl --insecure https://owl.fish.washington.edu/halfshell/genomic-databank/hemat_transcriptome_v1.5.fasta`

The problem has to do with the server certificate which is installed on that server (owl). Removing the `s` from `https` eliminates the certificate requirement when fetching files. Alternatively, telling `curl` that the site is "insecure" tells `curl` to ignore any certificate junk.

Sorry for the hassle!
